site/styles/_meta-detail.scss

```diff
@@ -186,6 +186,11 @@
       grid-row: 2;
     }
   }
+
+  .page-collection-list {
+    grid-row: 3;
+    grid-column-end: -1;
```

Comment:
Run the linter please!

Reply:
Also, I think `grid-column: 2;` is clearer than `grid-column-end: -1;` since we know there are two columns by design.  Cool trick though... I didn't know about negative grid values!

Reply:
It's on grid-column 1 under the lg breakpoint (you'll see the layout breaks).

It's pretty neat and saves you specifying grid-column-start as it grows to take up the entire space. Essentially flex-grow 1.

Reply:
Hold up I've confused myself thinking this CSS is shared by collections on the film detail page (where I would have been wrong about the column also lol).  Looks like collections on the film detail page live outside of `meta-detail-main`... would it be better to change the bundle page structure to match the film page?

Reply:
I tried it out and it seems like a good idea for consistency.

Only thing it needs the padding from swiper-container class so I'll wait for the collection-container PR merge

Reply:
@PeculiarGoat padding added, ready for test and approval